test/api/unit/authorizers/site.test.js

```diff
@@ -1,216 +1,272 @@
-const crypto = require("crypto")
-const expect = require("chai").expect
-const factory = require("../../support/factory")
+const crypto = require('crypto');
+const { expect } = require('chai');
 const nock = require('nock');
 const sinon = require('sinon');
+const factory = require('../../support/factory');
 const githubAPINocks = require('../../support/githubAPINocks');
-const authorizer = require("../../../../api/authorizers/site.js")
+const authorizer = require('../../../../api/authorizers/site.js');
 const siteErrors = require('../../../../api/responses/siteErrors');
 const FederalistUsersHelper = require('../../../../api/services/FederalistUsersHelper');
 
-describe("Site authorizer", () => {
-  describe(".create(user, params)", () => {
-    it("should resolve", done => {
+describe('Site authorizer', () => {
+  describe('.create(user, params)', () => {
+    beforeEach(() => {
+      Promise.all([
```

Comment:
I think these should actually return the promise https://mochajs.org/#working-with-promises